src/form-elements/FormElementSection.tsx

```diff
@@ -160,13 +171,36 @@ function FormElementSection<T extends FormTypes._NestedElementsElement>({
           hidden: 'ob-section__collapsed',
         }}
       >
-        <OneBlinkFormElements
-          {...props}
-          displayValidationMessages={displayValidationMessage}
-          onLookup={handleLookup}
-          elements={element.elements}
-          onUpdateFormElements={handleUpdateNestedFormElements}
-        />
+        {element.canCollapseFromBottom ? (
```

Comment:
My opinion: better to create a component that wraps OneBlinkFormElements and either renders the buttons and {children} or just {children}.  I can see someone coming along later and updating the top instance of OneBlinkFormElements and not the bottom one.